src/freenet/clients/http/FProxyToadlet.java

```diff
@@ -15,8 +16,11 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Date;
+import java.util.HashMap;
 import java.util.HashSet;
+import java.util.LinkedHashMap;
```

Comment:
Is this import actually used? I can’t locate it anywhere in the changes for this file…

Reply:
No, it isn’t … along with some more imports. Optimized the imports.